res/adjectives.txt

```diff
@@ -41,8 +41,10 @@ non-standard
 cached
 stored
 formatted
-assembling
-disassembling
 distributed
 virtual
 augmented
+binary
+classical
```

Comment:
Questionable compatibility; feels out-of-place and not commonly used in tech jargon